### DISCUSSION TITLE
Any way to integrate Riffusion into this audio/music UI?

### DISCUSSION THREAD

**Discussion by severian42** [GrandaddyShmax/audiocraft_plus#35] [Ideas]
Hey! First off, this interface and functionality is INCREDIBLE! Thank you so much for putting this together! I've been able to do some incredible stuff with my Mac Studio. I was wondering if there might be some way to incorporate Riffusion and it's various capabilities into this platform? Multiple attempts have been made by me on my local server to combine the Automatic111 extensions and such into this platform, but truth be told I am a total beginner and keep coming up short! 

I am a Professional Audio Engineer who's just been learning all of this within the past 6 months. Essentially, I've been trying to build and piece together a true and dedicated Gradio training platform for pre-trained and custom audio/music models. Any thoughts on the possibility of this or maybe a better way to make it happen? I'm willing to help out in any way I can and even throw some money at the problem if it helps!

Thanks for taking the time to read this : )

**Comment by GrandaddyShmax** [comment]
Not sure if I myself alone would be able to achieve making an all in one music generation tool with other AI tools. as of right now, my goal is to focus on audiocraft, perhaps later down the line I'll try heading towards that type of project.

Sorry for taking long time, I honestly forgot to check Discussions tab 😅
